--- original/gml_Object_o_overgame_Create_0.gml
+++ modified/gml_Object_o_overgame_Create_0.gml
@@ -1,5 +_,6 @@
 randomize()
 global.version = "0.1.4 Alpha"@644
+global.droolVersion = "I.DROOL 0.1.0 b1"@2395
 hardmodeUnlocked = load_data("hardmodeUnlocked"@251, 0)
 challengesUnlocked = load_data("challengesUnlocked"@254, 0)
 loopRecord = load_data("loopRecord"@391, 0)
@@ -10,11 +_,14 @@
 global.gamestarttimer = -1
 global.soundSocialTwitter = 0
 global.soundSocialDiscord = 0
+global.soundSocialDiscord2 = 0
 global.pauseSounds[8] = 0
+global.pauseSounds[9] = 0
 global.restartTimer = 0
 global.gameMode = 0
 global.challengeType = 0
 global.firstDeathEver = load_data("firstDeathEver"@645, 1)
+global.showDroolOptions = 0
 load_language()
 global.savedLevel = 1
 global.savedLoop = 1
