--- original/gml_Object_o_overgame_Create_0.gml
+++ modified/gml_Object_o_overgame_Create_0.gml
@@ -1,5 +_,5 @@
 randomize()
-global.version = "0.1.4 Alpha"@644
+global.version = "b1, 0.1.4 Alpha"@644
 hardmodeUnlocked = load_data("hardmodeUnlocked"@251, 0)
 challengesUnlocked = load_data("challengesUnlocked"@254, 0)
 loopRecord = load_data("loopRecord"@391, 0)
@@ -10,6 +_,7 @@
 global.gamestarttimer = -1
 global.soundSocialTwitter = 0
 global.soundSocialDiscord = 0
+global.soundSocialDiscord2 = 0
 global.pauseSounds[8] = 0
 global.restartTimer = 0
 global.gameMode = 0
